### PR TITLE
Update README files with local dev environment setup

### PR DESCRIPTION
**Pull request by hanifn**
## Description

This PR updates the README files with instructions on how to set up a local development environment using `wp-env`.

## Steps to Test

N/A

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,448 +1,475 @@
...
-* Email Notifications (on post status change and editorial comment)
-* Additional Post metadata
-* Quick Pitch Dashboard widget
-* Bug fix: sorting issue on Manage Posts page (Mad props to David Smith from Columbia U.)
-* Other bug fixes
-* Better localization support
-
-= 0.1.5 =
-* Ability to assign custom statuses to posts
+=== Edit Flow ===
```
Comment (ingeniumed) [inline readme.txt:1]: I had this happen as well, and the way I avoided the entire file changing was by doing cmd+k followed by s in vscode. That way it didn't format this file. Prob worth adding it to the prettierignore maybe

**Review by ingeniumed — COMMENTED** [review]
Could you make [this](https://github.com/Automattic/Edit-Flow/pull/727#discussion_r1625659915) change in your PR as well? Essentially, you'd need to:

- Change GPL2+ to GPL3+ in `composer.json`
- Add a LICENSE file at the root of the repo which is the GPL3 license

Also, I'll change your branch base to develop as we merge our changes in there first before cutting a release for master.

**Comment by hanifn** [comment]
> Could you make https://github.com/Automattic/Edit-Flow/pull/727#discussion_r1625659915 change in your PR as well?

@ingeniumed Updated the license in 22551d372fe3bdcdc60009a5806059d923e2eff0

File: readme.txt (hunk 1 above)
Comment (hanifn) [reply readme.txt:1]: I've tried reverting the changes and doing what you suggested but git is saying there are no changes to commit 🤔

**Comment by hanifn** [comment]
> Also, I'll change your branch base to develop as we merge our changes in there first before cutting a release for master.

@ingeniumed I've changed the target branch as well

**Review by ingeniumed — APPROVED** [review]

File: readme.txt (hunk 1 above)
Comment (ingeniumed) [reply readme.txt:1]: On second thoughts, lets leave it so we don't need to keep doing this revert. Better to get it done in one go

**Pull request merged**